firefox_ui_tests/remote/security/test_green_larry.py

```diff
@@ -0,0 +1,115 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected
+from marionette_driver.errors import NoSuchElementException
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestGreenLarry(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://ssl-ev.mozqa.com/'
+
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        try:
+            self.identity_popup.close(force=True)
+            self.windows.close_all([self.browser])
+        except NoSuchElementException:
+            # TODO: A NoSuchElementException is thrown here when the identity popup
+            # closes earlier, apparently when the page info window opens successfully
```

Comment:
You can remove the reference to the page info window. It's not related here given that it will also fail when the test method is skipped.